sbi/inference/snpe/snpe_base.py

```diff
@@ -166,6 +163,7 @@ def train(
         discard_prior_samples: bool = False,
         retrain_from_scratch_each_round: bool = False,
         show_train_summary: bool = False,
+        **dataloader_kwargs: Any,
```

Comment:
couldn't you make this an explicit kwarg of `train()`? E.g., `dataloader_kwargs: Optional[dict] = None,` and then the same in the `snpe_c` `train` method?

Reply:
in `snpe_c` line 108 add the same additional kwarg: `dataloader_kwargs: Optional[dict] = None`. I haven't tested it, just an idea.